server/src/repositories/job.repository.ts

```diff
@@ -227,19 +230,12 @@ export class JobRepository {
     return this.moduleRef.get<Queue>(getQueueToken(queue), { strict: false });
   }
 
-  public async removeJob(jobId: string, name: JobName): Promise<IEntityJob | undefined> {
-    const existingJob = await this.getQueue(this.getQueueName(name)).getJob(jobId);
-    if (!existingJob) {
-      return;
-    }
-    try {
+  // deprecated
```

Comment:
```suggestion
  /** @deprecated */
```

jsdoc comment with `@deprecated` will make it show up with strikethrough and on tooltip